PatentsView/PatentsView-DB
# PV-15 Fix

Test Cases file: 'tests/government_interest/vafix/parsed_data/TestData/government_interest.csv'
	File constructed of user data sent in as example of problematic data set.
Assumes 'tests' directory is in project root.

Test Cases execution instruction:
1. Update [FOLDERS] section of 'config.ini' in project root. WORKING_FOLDER should point to 'tests/government_interest/vafix'.
2. Run 'tests/government_interest/vafix/test_VAfix.py'.
3. Manually inspect 'tests/government_interest/vafix/government_interest/pre_manual/NER_out/edge_cases.csv' to verify favorable changes in solid matches. The 'org' column shows the organization name being matched, the 'with_space' column shows the solid matches for the organization with the new existing lookup logic, and the 'wo_space' column shows the solid matches for the organization with the old existing lookup logic.

Results:
Almost all results are unambiguously favorable, removing incorrectly matches or adding correct matches. There are some instances where an incorrect match is added or a correct match is removed, but these are few and being tagged for further investigation.

Proof of successful execution:
 
![image](https://user-images.githubusercontent.com/45615296/99563646-77b3ba80-2997-11eb-9181-82e1d16df209.png)


Example output:
 
![image](https://user-images.githubusercontent.com/45615296/99563664-7c786e80-2997-11eb-8ede-858dc03ea53e.png)
